src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinInetProxyHelper.cs

```diff
@@ -178,5 +181,42 @@ public bool ManualSettingsOnly
 
             return useProxy;
         }
+
+        public bool AutoDetectScriptSuccess()
```

Comment:
"AutoDetectScriptSuccess" -> "DetectAutoProxyConfig" is a better name.  Usually want <verb><noun> formats for these kind of method names that do an action and return a bool.